### DISCUSSION TITLE
数学公式怎么用不了哇

### DISCUSSION THREAD

**Discussion by Justsenger** [hugo-next/hugo-theme-next#97] [General]
已开启math: mathjax，尝试$\lfloor x \rfloor$并未正确渲染，页面显示还是$\lfloor x \rfloor$。

**Comment by elkan1788** [comment]
请提供下你的具体站点仓库代码，帮你看下具体原因。

**Comment by Laplace825** [comment]
小白同问,使用主题后没有更改js、css等文件,这里是我的站点仓库链接https://github.com/Aphrosmeteorite/aphrosmeteorite.github.io

**Reply by elkan1788** [reply]
啥意思？看你的站点似乎没有问题，也没有看到数据公式的文章。

**Reply by Laplace825** [reply]
大概是小端存储那篇,我使用了$(FFCFAF41)_H$类似这样的行内公式,但是页面显示依然是$(FFCFAF41)_H$,但如果$$(FFCFAF41)_H$$则可以正确显示,但是会独占一行

**Reply by JennySong924** [reply]
同问

**Comment by elkan1788** [comment]
@Laplace825 @JennySong924 
已经修复，现在可以支持行内数学公式嵌套，示例的效果如下：

[数学公式渲染](https://preview.hugo-next.eu.org/demo/math-formula.html#more)

**Comment by Justsenger** [comment]
可惜了，现在网站都关闭了，过于孤芳自赏。

**Reply by elkan1788** [reply]
哈， githubpage服务还是很方便的，你可以拥有。

**Reply by Justsenger** [reply]
主要是写了没人看😭
